CONTRIBUTING.md

```diff
@@ -1,79 +1,27 @@
-# Contributing to `dbt`
+# Contributing to `dbt-core`
+
+`dbt-core` is Apache 2.0-licensed open source software. `dbt-core` is what it is today because community members have opened issues, provided feedback, and [contributed to the knowledge loop](https://www.getdbt.com/dbt-labs/values/). Whether you are a seasoned open source contributor or a first-time committer, we welcome and encourage you to contribute code, documentation, ideas, or problem statements to this project.
```

Comment:
We already have a `License.md` file, do we need to call out the license here as well?

Reply:
Fair! This is us taking some extra credit for the fact that Apache 2.0 is one of the most permissive OSS licenses. I think this could just say "`dbt-core` is open source software. It is what it is today because..."